src/python/espressomd/visualization_opengl.pyx

```diff
@@ -345,8 +345,7 @@ class openGLLive:
         # OVERWRITE WITH USER PROPERTIES
         for key in kwargs:
             if key not in self.specs:
-                raise ValueError(
-                    key + ' is not a valid visualization property')
+                raise ValueError(f'{key} is no valid visualization property')
```

Comment:
f-strings are only available starting from Python 3.6 ([PEP 498](https://www.python.org/dev/peps/pep-0498/)). I think we should bump espresso to python 3.6 ASAP, python 3.5 was dropped from NEP 29 two months ago ([NEP 29 support table](https://numpy.org/neps/nep-0029-deprecation_policy.html#support-table)).

Reply:
Bumping python will conflict with the currently opened PR on CMake refactor... Since no test fails here, we can probably keep these f-strings and bump python in a few days in a separate PR, where we could also bump cython and python modules accordingly.

Reply:
the raise of the ```ValueError``` is probably untested.

Reply:
the visualizer is not tested. Our CI has python 3.5 only in the clang-6 container, but it cannot go away until we deal with walberla to rely on clang-9 instead of clang-6.